tss-esapi/src/context/general_esys_tr.rs

```diff
@@ -85,6 +85,26 @@ impl Context {
         Ok(())
     }
 
+    #[cfg(has_esys_tr_get_tpm_handle)]
+    /// Retrieve the `TpmHandle` stored in the given object.
+    pub fn tr_get_tpm_handle(&mut self, object_handle: ObjectHandle) -> Result<TpmHandle> {
+        use crate::tss2_esys::Esys_TR_GetTpmHandle;
+        use tss_esapi_sys::TPM2_HANDLE;
+        let mut tpm_handle: TPM2_HANDLE = Default::default();
```

Comment:
I didn't see a way to create a default `TpmHandle`, so I resorted to using the underlying type here. If there's a way to do this using only `TpmHandle`, that would certainly be an improvement.

Reply:
I am not sure. You could probably find some ```null``` that you can initialize it to and then make sure it has changed.

Reply:
Maybe ```TPM_RH_UNASSIGNED``` but I am not sure.

Reply:
Yeah I think that it will work because it is an internal value to the TPM that should never exist outside of it. So it is possible to add a check to see if the value has changed and if it has not then perhaps return an error.

Reply:
Agree, though I have no preference either way whether that check should be done - don't think we specifically verify for potential bugs in TSS everywhere we could.

Reply:
I think that kind of general error checking should be done in the `TpmHandle::try_from` method. Currently that method is returning an Ok result when given `TPM2_RH_UNASSIGNED`, perhaps we should modify it in a separate branch?